dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/CollectionUtils.java

```diff
@@ -98,8 +98,8 @@ public static boolean isEmpty(Collection coll) {
         }
         String[] strings = str.split(separator);
         Map<String, String> map = new HashMap<>(strings.length);
-        for (int i = 0; i < strings.length; i++) {
-            String[] strArray = strings[i].split("=");
+        for (String string : strings) {
```

Comment:
I do not think foreach is better than fori ,especial for []

Reply:
> I do not think foreach is better than fori ,especial for []

When he doesn’t use the index, it’s not so elegant to use this form, what do you think?

Reply:
I doesn't think so .Using index to traverse array [] is a standard way ,right ? Why is it not elegant?
foreach is better than fori when traversing List, Set, Array ,etc.